cmd/internal/get/traces.go

```diff
@@ -0,0 +1,136 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/akitasoftware/akita-cli/cmd/internal/akiflag"
+	"github.com/akitasoftware/akita-cli/printer"
+	"github.com/akitasoftware/akita-cli/rest"
+	"github.com/akitasoftware/akita-cli/util"
+	"github.com/akitasoftware/akita-libs/akid"
+	"github.com/akitasoftware/akita-libs/akiuri"
+	"github.com/akitasoftware/akita-libs/tags"
+)
+
+var GetTracesCmd = &cobra.Command{
+	Use:          "traces [AKITAURI|SERVICE]",
+	Aliases:      []string{"trace"},
+	Short:        "List traces for the given service.",
+	Long:         "List traces in the Akita cloud, filtered by service and by tag.",
+	SilenceUsage: false,
+	RunE:         getTraces,
+}
+
+func init() {
+	Cmd.AddCommand(GetTracesCmd)
+
+	GetTracesCmd.Flags().StringVar(
+		&serviceFlag,
+		"service",
+		"",
+		"Your Akita service.")
+
+	GetTracesCmd.Flags().StringSliceVar(
+		&tagsFlag,
+		"tags",
+		[]string{},
+		"Tag set to filter on, specified as key=value matches. Uses OR of tags.")
```

Comment:
The endpoint being called implements conjunction, not disjunction.

Reply:
Fixed in https://github.com/akitasoftware/akita-cli/pull/65/commits/058c7e8a85f7d431f9d51374377310cef5af211a